src/test/rustdoc-ui/no-crate-level-doc-lint.rs

```diff
@@ -0,0 +1,3 @@
+#![deny(missing_crate_level_doc)]
```

Comment:
```suggestion
#![deny(missing_crate_level_docs)]
```
or better
```suggestion
#![deny(missing_crate_docs)]
```
We have naming conventions for lints - https://github.com/rust-lang/rfcs/blob/master/text/0344-conventions-galore.md#lints.